bindings/pyroot/cppyy/cppyy-backend/clingwrapper/src/clingwrapper.cxx

```diff
@@ -1714,13 +1714,30 @@ Cppyy::TCppMethod_t Cppyy::GetMethodTemplate(
     return (TCppMethod_t)nullptr;
 }
 
+static inline
+std::string type_remap(const std::string& n1, const std::string& n2)
```

Comment:
Wow, this is hacky, isn't it? But I also cannot make any better proposal.
Do we have tests for this?

Reply:
Tests are coming!